pandas/core/arrays/numeric.py

```diff
@@ -199,3 +201,48 @@ def reconstruct(x):
             return tuple(reconstruct(x) for x in result)
         else:
             return reconstruct(result)
+
+    def _apply(self, func: Callable, **kwargs) -> NumericArray:
+        """
+        Applying a given function on the array
+
+        Parameters
+        ----------
+        func : callable function to apply
+        **kwargs
+            additional arguments for the function if necessary
+
+        Returns
+        -------
+        NumericArray
+            The image of the NumericArray
+        """
+        values = func(self._data, **kwargs)
+        return type(self)(values, self._mask.copy())
+
+    def round(self, decimals: int = 0, *args, **kwargs) -> NumericArray:
```

Comment:
is *args needed? having *args after a keyword argument can be problematic

Reply:
I did the same as already done in some other files of the library. For instance [here](https://github.com/pandas-dev/pandas/blob/dfb92d0f1bca79185128538698fbc660077eccfd/pandas/core/frame.py#L8393)